platform/broadcom/sonic-platform-modules-nexthop/debian/sonic-platform-nexthop-4010.postinst

```diff
@@ -0,0 +1,13 @@
+depmod -a
```

Comment:
Its always a good idea to add a ".prerm" file for the debian along with ".postinst". That will help in de-init everything while removing the platform pkg.